tests/config_test.py

```diff
@@ -80,8 +80,17 @@ def test_configure_from_object_changes_scribe_disable(self):
                 '    scribe_disable = False\n'
                 'clog.config.configure_from_object(C)\n'
                 'print(clog.config.scribe_disable)\n'
-            ),
-            stdout=subprocess.PIPE,
-        )
-        out = proc.communicate()[0].decode('UTF-8')
+            )).decode('UTF-8')
         assert out == 'True\nFalse\n'
+
+    def test_logging_not_configured(self):
```

Comment:
Can you additionally test that logging does work when configured via the 3 different methods?